lls/main.c

```diff
@@ -789,12 +895,18 @@ lls_stage2(void *arg)
 		ret = register_nd_acl_rules(&net_conf->front);
 		if (ret < 0)
 			return ret;
+		ret = register_nd_router_sol_or_adv_acl_rules(&net_conf->front);
+		if (ret < 0)
+			return ret;
 	}
 
 	if (lls_conf->nd_cache.iface_enabled(net_conf, &net_conf->back)) {
 		ret = register_nd_acl_rules(&net_conf->back);
 		if (ret < 0)
 			return ret;
+		ret = register_nd_router_sol_or_adv_acl_rules(&net_conf->front);
```

Comment:
This second call of `register_nd_router_sol_or_adv_acl_rules()` should pass `&net_conf->back`.